### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by rudicarrilloypr**
- [x] - Delete all the boilerplate from Create React App (text, images, styles).
- [x] - Create a new directory called components.
- [x] - Inside components, create a new Calculator.js file.
- [x] - In Calculator.js, create a React component that matches this design: Calculator
- [x] - Implement the design only (JSX and CSS)
- [x] - Develop just the UI of the calculator; the math logic to make it work will be added in the next project.
- [x] - Use functional components
- [x] - Create at least 1 child component
- [x] - Once the Calculator is ready, import it and use it in your top-level component: App.jsx
- [x] - Test it in the browser; it should render without problems.

### CODE REVIEW

File: src/components/Calculator.jsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+  return <button type="button" data-type={type} data-value={value}>{value}</button>;
+}
+
+Button.propTypes = {
+  value: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.number,
+  ]).isRequired,
+};
```
Comment (Ismailco) [inline src/components/Calculator.jsx]: - [x] Kindly make sure to add the code for the `Button` component in a separate file and import it to the `Calculator` component instead

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @rudicarrilloypr, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
- [x] Kindly make sure to remove all the boilerplate code generated from CRA, including images, icons, and script files that are not necessary for your code

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.jsx (hunk 1 above)
Comment (rudicarrilloypr) [reply src/components/Calculator.jsx]: hi @Ismailco but why should I do that? that's not on the requirements, Kindly add some feedback to understand why are you asking me this...

Comment (Ismailco) [reply src/components/Calculator.jsx]: Hi @rudicarrilloypr, it is required to use 1 component inside the `Calculator`, also it's best practice to divide your code in React, one file should include one component

Comment (Whoistolu) [reply src/components/Calculator.jsx]: - [x] Well resolved 🥇

File: src/components/Calculator.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    <div className="calculator">
+      <div className="result">0</div>
+      {buttons.map((buttonValue) => (
+        <Button key={buttonValue} value={buttonValue} />
+      ))}
+    </div>
+  );
+}
+
+export default Calculator;
```
Comment (Whoistolu) [inline src/components/Calculator.jsx:18]: - [x] [OPTIONAL] Kindly ensure that the  `Calculator.js` component should match the given design as stated in the project requirement.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
Screenshot 2023-07-11 at 23 02 08 | Screenshot 2023-07-11 at 23 02 46

**Review by Whoistolu — APPROVED** [review]
### Hi @rudicarrilloypr 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

- [x]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.
- [x] [OPTIONAL] I am adding this recommendation here because you do not have it in the file changes. Let's avoid disabling Eslint as it helps us maintain a standard Javascript code. Kindly enable Eslint and manually fix the bugs.
Screenshot 2023-07-11 at 23 07 05


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**